docs/diagrams/Budget.puml

```diff
@@ -10,7 +10,7 @@ return ui
 
```

Comment:
For budget sequence diagram, you might want to add : for objects of classes or add <<class>> if you are interacting with class static methods

Reply:
fixed